### PR TITLE
Add company category

### PR DESCRIPTION
**Pull request by aldinaufal21**

### CODE REVIEW

File: test/categories/test_company.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class TestCompany(unittest.TestCase):
+    def setUp(self):
+        self.company = Company()
+
+    def test__get_values__is_valid(self):
+        error_message = CategoryHelper.get_values_error_message(self.company)
+        self.assertTrue(error_message == "", error_message)
+
```
Comment (aldinaufal21) [inline test/categories/test_company.py:13]: Sorry @gabrielbarker , I want to ask, I've changed test_company to be like this, but it's still failed, guess what's wrong with this code?

Comment (gabrielbarker) [reply test/categories/test_company.py:13]: Hey @aldinaufal21 , I recently added some tests that make sure all the values in the `get_values` method are in alphabetical order. There are details of the recent changes in the `contributing.md` file :)

**Comment by aldinaufal21** [comment]
sorry, i still don't understand what is explained in contribution.md. for code
from <YOUR_CATEGORY_FILE> import <YOUR_CATEGORY>
print (sorted (<YOUR_CATEGORY> () .get_values ()))
where is it written?

**Comment by gabrielbarker** [comment]
@aldinaufal21 The values in `get_values` in company.py need to be alphabetical
The code in contributing tells you how to quickly alphabetise a list. i.e.
- Open your command line
- Type `python`
- Then copy and paste 
```
from qual_id.categories.company.py import Category
print (sorted (Category() .get_values ()))
```
The terminal will then print the alphabetised list. I thought it might be helpful if people have big lists that it would take a long time to sort

Also, if you click on `Details` next to the build failed box on this page, and then click `the build` you will be able to see the jobs ran for this PR. If you click one of the jobs and go to the `job log`, at the bottom you can see the test failures. These should tell you why they have failed and in what file the error occured 🙂

**Review by gabrielbarker — APPROVED** [review]
Thanks for making these changes

**Comment by gabrielbarker** [comment]
Thanks a lot @aldinaufal21 !🙂

**Pull request merged**